File: src/main/java/com/aquatrack/service/ResidentWaterBillService.java
package com.aquatrack.service;

import com.aquatrack.dto.resident.ResidentWaterBillResponseDto;

import java.util.List;

public interface ResidentWaterBillService {

    // ==========================================
    // Get My Water Bills
    // ==========================================

    List<ResidentWaterBillResponseDto> getMyWaterBills();

}

File: Aquatrack-frontend/src/services/residentWaterBillService.js
import api from "../api/axios";

const BASE_URL = "/resident/water-bills";

const residentWaterBillService = {

    // ==========================================
    // Get My Water Bills
    // ==========================================

    getMyWaterBills: async () => {

        const response = await api.get(BASE_URL);

        return response.data.data;

    }

};

export default residentWaterBillService;